shelly/src/main/java/au/com/jtribe/shelly/Email.java

```diff
@@ -130,7 +129,7 @@ public boolean send() {
         }
 
         if (emailIntent.resolveActivity(this.context.getPackageManager()) != null) {
-            this.context.startActivity(emailIntent);
+            this.context.startActivity(Intent.createChooser(emailIntent,"Share email via:"));
```

Comment:
I'm wary of passing in a title to `createChooser`, because it won't be translated.

If we were going to offer support for a title for the chooser we would need to offer an API for setting it with a null as a default.